src/speakers/components/mobile-filters.tsx

```diff
@@ -195,7 +125,7 @@ const MobileFilters = (): ReactElement => {
                 <Box display="flex" justifyContent="space-around" marginTop="40px">
```

Comment:
display="flex" justifyContent="space-around" marginTop="40px"

should be in CSS